### PR TITLE
Adding extra globals

### PR DESCRIPTION
**Pull request by fabiomcosta**
including some browser builtins and some JS globals

### CODE REVIEW

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js [hunk 1]
```diff
@@ -17,24 +17,51 @@ module.exports = new Set([
   'Boolean',
   'Date',
   'Error',
+  'Function',
+  'Infinity',
   'JSON',
   'Math',
   'Number',
   'Object',
+  'Promise',
```
Comment (kyldvs) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: I guess this is fine now to not require in www, right @DmitrySoshnikov ?

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js [hunk 2]
```diff
@@ -17,24 +17,51 @@ module.exports = new Set([
...
+  'performance',
+  'prompt',
+  'screen',
+  'sessionStorage',
+  'setInterval',
+  'setTimeout',
   'window',
+  
+  'Option',
+  'Image',
```
Comment (kyldvs) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: I think this one will conflict with `Image.react` which we do want to require. Can we remove this?

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js [hunk 3]
```diff
@@ -17,24 +17,51 @@ module.exports = new Set([
...
+  'open',
+  'performance',
+  'prompt',
+  'screen',
+  'sessionStorage',
+  'setInterval',
+  'setTimeout',
   'window',
+  
+  'Option',
```
Comment (kyldvs) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: What is `Option`?

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js (hunk 1 above)
Comment (DmitrySoshnikov) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: Yep, should be global now (auto-imported module).

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js (hunk 3 above)
Comment (DmitrySoshnikov) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: https://developer.mozilla.org/en-US/docs/Web/API/HTMLOptionElement/Option

File: pkg/nuclide/format-js-base/lib/constants/builtIns.js (hunk 2 above)
Comment (fabiomcosta) [inline pkg/nuclide/format-js-base/lib/constants/builtIns.js]: :+1:

**Comment by kyldvs** [comment]
LGTM, someone else will need to merge it in

**Comment by fabiomcosta** [comment]
I forgot `navigator` :|

**Pull request closed without merging**